commands/watch.go

```diff
@@ -42,7 +42,7 @@ var WatchCmd = &cli.Command{
 	Usage: "Start a daemon job to watch the head of the filecoin blockchain.",
 	Flags: []cli.Flag{
 		&cli.IntFlag{
-			Name:        "indexhead-confidence",
+			Name:        "confidence",
```

Comment:
You'll need to coordinate this change in the helm chart: https://github.com/filecoin-project/helm-charts/blob/master/charts/visor/templates/deployment.yaml#L90. Maybe include a conditional error that throws if the requested version of visor doesn't work with the new chart changes?

Reply:
This is just for the lily version of the command, the standard one is left unchanged